src/linux/system.rs

```diff
@@ -269,6 +272,19 @@ impl SystemExt for System {
                     }
                 }
             }
+
+            // Linux < 3.14 may not have MemAvailable in /proc/meminfo
+            // So it should fallback to the old way of estimating available memory
+            // https://github.com/KittyKatt/screenFetch/issues/386#issuecomment-249312716
+            if self.mem_available.is_none() {
```

Comment:
This will never be `None` once we got the information once, preventing the refresh to work as expected, right? I think instead of using a `Option`, you should instead add a boolean in here saying whether or not `MemAvailable` was found. Does that sound good to you?

Reply:
Uh, I forgot about that. I'll update the PR tomorrow, it's already night in my time zone.

Reply:
No problem, thanks for the contribution! Waiting for your update then. :)